mock_Issues.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import "github.com/stretchr/testify/mock"
+
+import "github.com/google/go-github/github"
+
+type MockIssues struct {
+	mock.Mock
+}
+
+func (_m *MockIssues) AddLabelsToIssue(owner string, repo string, number int, labels []string) ([]github.Label, *github.Response, error) {
```

Comment:
The `mock_` files are generated with [mockery](https://github.com/vektra/mockery) and are not meant to be manually modified. I have a cleanup PR coming up where I can try moving these mocks to a separate folder so it's obvious that it's generated code.